api/__mocks__/uuid/v4.js

```diff
@@ -0,0 +1,5 @@
+function uuidv4() {
+  return 'mocked uuid'
+}
+
+module.exports = uuidv4
```

Comment:
this is how you mock a module - this way in testing the `uuidv4()` function returns `'mocked uuid'` when it's called in the mutation rather than a random uuid. 🎉